### ISSUE TITLE
'turbo run' should prompt you which packages you'd like to run

### ISSUE THREAD

**Issue by mattpocock** [vercel/turborepo#1237]
### Describe the feature you'd like to request

1. User runs `turbo run`
2. Using `prompts` (multiselect) or similar, `turbo` looks at the available scripts and asks which ones you want to run

### Describe the solution you'd like

See above

### Describe alternatives you've considered

The current behaviour (saying 'you must specify at least one script') is fine, but there's an opportunity for a cooler flow here.

**Comment by jaredpalmer** [comment]
Agreed. I want to drop "run" or make it optional as well, but I also like this for the empty case.

**Comment by ragrag** [comment]
would be great to have this functionality built into turbo!
i built a tiny package to do that (prompts for script and package filters)
https://github.com/ragrag/turbo-select
